test/common/stats/thread_local_store_test.cc

```diff
@@ -65,7 +67,7 @@ class HistogramTest : public testing::Test, public RawStatDataAllocator {
     }));
 
     EXPECT_CALL(*this, alloc("stats.overflow"));
-    store_.reset(new ThreadLocalStoreImpl(*this));
+    store_.reset(new ThreadLocalStoreImpl(options_, *this));
```

Comment:
nit: `std::make_unique`

Reply:
nit: can we use `std::make_unique` here?

Reply:
Done.

Reply:
Done.